feat(useAverage): new function (#1826)
vueuse/vueuse: packages/math/useAverage/index.md

---
category: '@Math'
---

# useAverage

> Get the average value of a branch of numbers

Reviewer: ```suggestion
Get the average value of an array of numbers
```
